sdk/textanalytics/Azure.AI.TextAnalytics/src/AnalyzeSentimentResultCollection.cs

```diff
@@ -7,6 +7,9 @@
 namespace Azure.AI.TextAnalytics
 {
     /// <summary>
+    /// Collection of <see cref="AnalyzeSentimentResult"/> entities corresponding
```

Comment:
Should we be using `entities` here?

Reply:
Nice catch!  I changed it in other places and will update here as well.  Thanks!